### PR TITLE
INTYGFV-15767: Updated icons for global alert

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

File: apps/rehabstod/src/components/Layout/Layout.tsx [hunk 1]
```diff
@@ -15,7 +15,9 @@ export function Layout() {
     <div className="flex min-h-screen flex-col">
       <LayoutHeader />
       <main className="flex-1">
-        {config && config.banners.length > 0 && <GlobalAlert>{config.banners[0].message}</GlobalAlert>}
+        {config && config.banners.length > 0 && (
+          <GlobalAlert priority={config.banners[0].priority}>{config.banners[0].message}</GlobalAlert>
+        )}
```
Comment (SamuelTilly) [inline apps/rehabstod/src/components/Layout/Layout.tsx:20]: Should we only display the first available banner if there are more than one? Looking at the old client's code; it looks like it displays them all.

File: apps/rehabstod/src/components/GlobalAlert/GlobalAlert.tsx [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-import { IDSAlertGlobal, IDSIconAttention } from '@frontend/ids-react-ts'
+import { IDSAlertGlobal, IDSIconAttention, IDSIconInformation, IDSIconWarning } from '@frontend/ids-react-ts'
+import { BannerPriority } from '../../schemas'
 
-export function GlobalAlert({ children }: { children: ReactNode }) {
+export function GlobalAlert({ children, priority }: { children: ReactNode; priority: BannerPriority }) {
   return (
     <IDSAlertGlobal headline="Driftmeddelande">
-      <IDSIconAttention title="informationsikon" />
+      {priority === BannerPriority.LOW && <IDSIconInformation data-testid="LOW_ICON" />}
```
[the remark this answers is not in the document]
Comment (mhornfeldt) [reply apps/rehabstod/src/components/GlobalAlert/GlobalAlert.tsx:8]: This doesn't work for how the IDS-component is rendered. We need to have it inside the return statement and not in a function. From what I remember we can't have multiple ?:-expressions either.

File: apps/rehabstod/src/components/Layout/Layout.tsx (hunk 1 above)
Comment (mhornfeldt) [reply apps/rehabstod/src/components/Layout/Layout.tsx:20]: I can add it but test said that only one banner could be added at a time in IA and in the requirements meeting we also talked about that we don't add more than one and don't want to show more than one.

Comment (SamuelTilly) [reply apps/rehabstod/src/components/Layout/Layout.tsx]: Alright, we can keep it like this, was just curious :)

**Review by SamuelTilly — APPROVED** [review]

**Pull request merged**